source/_ecosystem/ios/notifications/content_extensions.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Dynamic content"
+description: "Extend your notifications with dynamic content"
+date: 2016-10-25 15:00:00 -0700
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+With the new Content Extension feature found in iOS 10, dynamic content can now be displayed as part of a notification without opening an app.
```

Comment:
Why don't we merge this page together with attachments and just make the page "Advanced Notifications" ?